code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -1,12 +1,11 @@
-// ALEPH armor, go wild, but attempt to keep total armor at ~245 total.
-// Paradise lost and Twilight are the only exceptions.
```

Comment:
Read this.

Reply:
They're still better than all the other aleph armor.